composer.json

```diff
@@ -169,7 +173,10 @@
         "drupal/simple_oauth": "2.0.0-rc3",
         "oomphinc/composer-installers-extender": "^1.1",
         "drupal/dropzonejs": "1.0.0-alpha7",
-        "bower-asset/dropzone": "^5.1"
+        "bower-asset/dropzone": "^5.1",
+        "drupal/crop": "1.2.0",
```

Comment:
I had to add Crop API as an explicit dependency here in order to patch it. Once the patch is committed, we can remove this dependency, since Image Widget Crop already depends on Crop API.